source/vir/src/datatype_to_air.rs

```diff
@@ -139,14 +139,16 @@ fn datatype_or_fun_to_air_commands(
         // box axiom:
         //   forall x. x == unbox(box(x))
         // trigger on box(x)
-        let bind = func_bind(ctx, &Arc::new(vec![]), &x_params(&datatyp), &box_x, false);
+        let name = format!("{}_box_axiom", path_as_rust_name(dpath));
```

Comment:
Please define this and similar ones in `def.rs`

Reply:
Done.